app/src/main/java/de/tum/in/tumcampusapp/api/app/ApiHelper.java

```diff
@@ -30,7 +30,8 @@
 import static de.tum.in.tumcampusapp.utils.Const.API_HOSTNAME_NEW;
 import static de.tum.in.tumcampusapp.utils.Const.STUDY_ROOMS_HOSTNAME;
 
-public final class Helper {
+public final class ApiHelper {
```

Comment:
A small explaining comment *which* API this helper is for would be nice

Reply:
It’s used in all API clients.